cpu/sam0_common/periph/cpuid.c

```diff
@@ -24,6 +24,7 @@
 #include <string.h>
 
 #include "periph/cpuid.h"
+#include "board.h"
```

Comment:
I'm not sure including a board specific header in a cpu file is a good idea.

Reply:
I have no preference so any idea to get around this issue is welcome :)